src/python/WMCore/WMSpec/Steps/Executors/CMSSW.py

```diff
@@ -273,7 +277,18 @@ def execute(self, emulator=None):
 
         os.environ.update(envOverride)
 
-        returnCode = subprocess.call(args, stdout=stdoutHandle, stderr=stderrHandle)
+        time0 = time.time()
+        # returnCode = subprocess.call(args, stdout=stdoutHandle, stderr=stderrHandle)
+        proc = psutil.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
I think this line breaks the current functionality as well. My understanding is that PIPE would make out/err logs to go to stdout instead of the specific files.

Reply:
it is resolved in new commit.